test/fixtures/experimental/class-private-properties/asi-failure-computed/actual.js

```diff
@@ -1,4 +1,4 @@
 class Foo {
   #p = x
-  [#m] () {}
```

Comment:
I think my example should throw, by looking at it again.

What I meant is this:
```js
class A {
  #x = 1 
  [this.#x] = 2
  //or
  #4 = this.#x
}
```

But these are also early errors?